### ISSUE TITLE
Don't output spaces on empty lines

### ISSUE THREAD

**Issue by Deraen** [kkinnear/zprint#130]
So I think this is separate to `:respect-bl` and other styles and options.

Given for example code:

```
(defn example-function [world]
  (foobar)

  (doseq [i (range 5)]
    (println)))
```

The output will contain two spaces on the empty line.

I think there probably is no reason to ever write trailing spaces. Or at least they shouldn't be added if they weren't present in the input?

**Comment by kkinnear** [comment]
Good catch!  I'm thinking that I probably should never write trailing spaces, regardless of whether they were there in the input or not.  I'll work on this, thanks for noticing this and letting me know!

**Comment by kkinnear** [comment]
This is fixed in 1.0.0.  I have several tests for this now.  If you find any problems, please let me know!
